sycl/test-e2e/Matrix/joint_matrix_bf16_fill_k_cache_unroll.cpp

```diff
@@ -6,6 +6,7 @@
 //
 //===----------------------------------------------------------------------===//
 // REQUIRES: matrix
+// XFAIL: cpu
```

Comment:
Could you please add comment below on why cpu was disabled for this test case?

Reply:
i put it in summary

Reply:
What I meant was that when reading code it is not clear, why fail is expected for CPU, so it is better to have comment about that in the code itself for future reference. But if you update the test as Dounia suggests, then this comment is not relevant.